projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery.php

```diff
@@ -130,26 +151,37 @@ public function get_attachments() {
 		return $attachments;
 	}
 
+	/**
+	 * Enqueue the default scripts and styles.
+	 */
 	public static function default_scripts_and_styles() {
 		wp_enqueue_script(
 			'tiled-gallery',
 			Assets::get_file_url_for_environment(
 				'_inc/build/tiled-gallery/tiled-gallery/tiled-gallery.min.js',
 				'modules/tiled-gallery/tiled-gallery/tiled-gallery.js'
 			),
-			array()
+			array(),
+			gmdate( 'YW' ),
```

Comment:
Should we use `JETPACK__VERSION` here?

```suggestion
			JETPACK__VERSION,
```